include/sway/tree/container.h

```diff
@@ -70,7 +70,9 @@ struct sway_container {
 	enum sway_container_layout prev_layout;
 	enum sway_container_layout workspace_layout;
 
-	// in output-local coordinates
+	// For C_ROOT, this has no meaning
+	// For C_OUTPUT, this is the output position in layout coordinates
+	// For other types, this is the position in output-local coordinates
```

Comment:
How about using a wlr_box while you're at it

Reply:
No, because HiDPI.